python/singa/image_tool.py

```diff
@@ -426,7 +426,33 @@ def random_crop(self, patch, inplace=True):
             return self
         else:
             return new_imgs
-
+        
```

Comment:
In random_crop the patch size has to be specified (as an input). This is the output image size.

But in random_size_crop the output size of cropped image is a random integer itself.